tests/backends/sqlalchemy/test_evaluate.py

```diff
@@ -31,7 +31,7 @@ class Record(Base):
             geometry_type="MULTIPOLYGON",
             srid=4326,
             spatial_index=False,
-            management=True,
+            # management=True,
```

Comment:
Pending outcome of #86 / #87

Reply:
The rebase introduced a commit which removed this parameter